dae/dae/backends/storage/impala_genotype_storage.py

```diff
@@ -0,0 +1,61 @@
+import os
+
+from dae.backends.storage.genotype_storage import GenotypeStorage
+
+from dae.backends.configure import Configure
+
+from dae.backends.impala.impala_helpers import ImpalaHelpers
+from dae.backends.impala.impala_variants import ImpalaFamilyVariants
+
+
+class ImpalaGenotypeStorage(GenotypeStorage):
+    
+    def __init__(self, storage_config):
+        super(ImpalaGenotypeStorage, self).__init__(storage_config)
+
+        self._impala_connection = self._make_impala_connection()
+
+    def _create_impala_connection(self):
+        connection = ImpalaHelpers.create_impala_connection(
+            self.storage_config.impala.host,
+            self.storage_config.impala.port
+        )
+
+        return connection
+
+    def get_backend(self, study_config, genomes_db):
+        impala_config = self._impala_configuration(study_config).impala
+        
+        variants = ImpalaFamilyVariants(
+            impala_config, self._impala_connection,
+            genomes_db.get_gene_models()
+        )
+
+        return variants
+
+    def _impala_configuration(self, study_config):
+        data_path = self._get_data_path(study_config.id)
+
+        if 'pedigree_file' in study_config:
+            pedigree_file = study_config.pedigree_file
+        else:
+            pedigree_file = "{}_pedigree.parquet".format(study_config.id)
+        if 'variant_files' in study_config:
+            variant_files = study_config.variant_files
```

Comment:
not sure what is this?? I believe the impala genotype storage should not have a notion of files??